### PR TITLE
test(journey): update helper funcs to use convo

### PR DESCRIPTION
**Pull request by mwegman**
- Instead of using the convo id, accept the whole conversation object.

SPARK-108103 #done

### CODE REVIEW

File: test/journeys/lib/convo.js [hunk 1]
```diff
@@ -1,119 +1,114 @@
...
+export function muteMessageNotification(spark, convo) {
   let activity;
 
-  browser.call(() => spark.internal.conversation.muteMessages({
-    id: convoId
-  }).then((a) => {
-    activity = a;
+  browser.call(() => spark.internal.conversation.muteMessages(convo)
+    .then((a) => {
+      activity = a;
```
Comment (InteractiveTimmy) [inline test/journeys/lib/convo.js]: Is there a purpose for this `then(() => {})` clause?

File: test/journeys/lib/convo.js [hunk 2]
```diff
@@ -1,119 +1,114 @@
...
+export function unmuteMentionsNotification(spark, convo) {
   let activity;
 
-  browser.call(() => spark.internal.conversation.unmuteMentions({
-    id: convoId
-  }).then((a) => {
-    activity = a;
+  browser.call(() => spark.internal.conversation.unmuteMentions(convo)
+    .then((a) => {
+      activity = a;
```
Comment (InteractiveTimmy) [inline test/journeys/lib/convo.js]: See my previous comment for this `then(() => {})` clause

File: test/journeys/lib/convo.js [hunk 3]
```diff
@@ -1,119 +1,114 @@
...
+export function muteMentionsNotification(spark, convo) {
   let activity;
 
-  browser.call(() => spark.internal.conversation.muteMentions({
-    id: convoId
-  }).then((a) => {
-    activity = a;
+  browser.call(() => spark.internal.conversation.muteMentions(convo)
+    .then((a) => {
+      activity = a;
```
Comment (InteractiveTimmy) [inline test/journeys/lib/convo.js]: See my previous comment for this `then(() => {})` clause

File: test/journeys/lib/convo.js [hunk 4]
```diff
@@ -1,119 +1,114 @@
...
-
-    return activity;
-  }).catch((e) => {
-    console.error('Unable to remove all mute tags');
-    console.error(e);
-    throw (e);
-  }));
+  browser.call(() => spark.internal.conversation.removeAllMuteTags(convo)
+    .then((a) => {
+      activity = a;
```
Comment (InteractiveTimmy) [inline test/journeys/lib/convo.js]: See my previous comment for this `then(() => {})` clause

**Comment by lalli-flores** [comment]
Agreed with @InteractiveTimmy, I would prefer the functions to return something like
```js
return broser.cal(...).catch(..)
```

**Pull request merged**